terraform/examples/io500/io500-sc22.config-template.daos-rf0.ini

```diff
@@ -33,26 +33,18 @@ run = TRUE
 
 [timestamp]
 
-[find-easy]
-nproc = $NPROC
-run = FALSE
-
 [ior-hard]
-API = DFS --dfs.pool=$DAOS_POOL --dfs.cont=$DAOS_CONT --dfs.dir_oclass=S1 --dfs.oclass=SX --dfs.chunk_size=470080
+API = DFS --dfs.pool=$DAOS_POOL --dfs.cont=$DAOS_CONT --dfs.chunk_size=470080 --dfs.dir_oclass=S1 --dfs.oclass=SX
 segmentCount = 10000000
-verbosity =
 run = TRUE
+verbosity =
 
 [mdtest-hard]
 API = DFS --dfs.pool=$DAOS_POOL --dfs.cont=$DAOS_CONT --dfs.dir_oclass=SX --dfs.oclass=S1
```

Comment:
Just for my understanding why is oclass and dir_oclass values are inverted between mdtest and ior ?

Reply:
I'm not sure. The file came from https://github.com/mchaarawi/io500/blob/main/config-full-isc22.ini which was provided in the instructions in https://daosio.atlassian.net/wiki/spaces/DC/pages/11167301633/IO-500+SC22

Reply:
Thanks for the link, I will look at it: could be useful for my performance tests with Azure.